packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.scss

```diff
@@ -2,138 +2,42 @@
 @use '../../../../style/typography';
 @use '../../../../style/mixins';
 
-gux-tab-advanced-list {
-  .gux-tab-container {
+.gux-tab-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  margin-bottom: 16px; //TODO
```

Comment:
Is this `margin-bottom` is still needed ?

Reply:
Yes I have added it to the token issue https://inindca.atlassian.net/browse/COMUI-2444